saleor/graphql/product/types/products.py

```diff
@@ -726,7 +726,9 @@ def resolve_products(root: models.ProductType, info, **_kwargs):
     @staticmethod
     @permission_required(ProductPermissions.MANAGE_PRODUCTS)
     def resolve_available_attributes(root: models.ProductType, info, **kwargs):
-        qs = models.Attribute.objects.get_unassigned_attributes(root.pk)
+        qs = models.Attribute.objects.get_unassigned_attributes(root.pk).filter(
```

Comment:
Maybe it will be useful to add querysets for different attribute types. E.g. `Attribute.objects.product_attributes` and `Attribute.objects.page_attributes`?